tests/state_test.go

```diff
@@ -95,6 +95,17 @@ func (suite *ExecutionSpecStateTestSuite) TestExecutionSpecState() {
 	st.skipLoad(`^berlin\/`)
 	st.skipLoad(`^cancun\/`)
 
+	// TODO-Kaia: should remove these skip
+	// json format error
+	st.skipLoad(`^prague\/eip7702_set_code_tx\/set_code_txs\/invalid_tx_invalid_auth_signature.json`)
+	st.skipLoad(`^prague\/eip7702_set_code_tx\/set_code_txs\/tx_validity_chain_id.json`)
+	st.skipLoad(`^prague\/eip7702_set_code_tx\/set_code_txs\/tx_validity_nonce.json`)
+	// EIP-3541 violation
+	st.skipLoad(`^prague\/eip7702_set_code_tx\/set_code_txs\/deploying_delegation_designation_contract.json`)
+
+	// tests to skip
+	st.skipLoad(`^prague\/eip2537_bls_12_381_precompiles\/`)
```

Comment:
> Only EIP-7702 is tested by Prague fork of Kaia because currently released test set for Prague has only prague EIPs.

I'm just wondering, does this test fail as of now?

Reply:
Yes. Is EIP-2537 supported on Kaia?

Reply:
Oh, we supported it at #111. I'll check it.